doc/reference/configuration/configuration_reference.rst

```diff
@@ -221,6 +488,718 @@ This section describes options related to :ref:`storing configuration in etcd <c
+
+
+.. _configuration_reference_database_txn_timeout:
+
+.. confval:: database.txn_timeout
+
+    A timeout (in seconds) after which the transaction is rolled back.
+
+    See also: :ref:`box.begin() <box-begin>`
+
+    | Type: number
+    | Default: 3153600000
+    | Environment variable: TT_DATABASE_TXN_TIMEOUT
+
+
+.. _configuration_reference_database_use_mvcc_engine:
+
+.. confval:: database.use_mvcc_engine
+
+    Whether the :ref:`transactional manager <txn_mode_transaction-manager>` is enabled.
+
+    | Type: boolean
+    | Default: false
+    | Environment variable: TT_DATABASE_USE_MVCC_ENGINE
+
+
+
+
+
+..  _configuration_reference_iproto:
+
+iproto
+------
+
+The ``iproto`` section is used to configure parameters related to communicating to and between cluster instances.
+
+.. NOTE::
+
+    ``iproto`` can be defined in any :ref:`scope <configuration_scopes>`.
+
+-   :ref:`iproto.advertise.* <configuration_reference_iproto_advertise>`
+
+    -   :ref:`iproto.advertise.client <configuration_reference_iproto_advertise_client>`
+    -   :ref:`iproto.advertise.peer <configuration_reference_iproto_advertise_peer>`
+    -   :ref:`iproto.advertise.sharding <configuration_reference_iproto_advertise_sharding>`
+
+-   :ref:`iproto.listen <configuration_reference_iproto_listen>`
+-   :ref:`iproto.net_msg_max <configuration_reference_iproto_net_msg_max>`
+-   :ref:`iproto.readahead <configuration_reference_iproto_readahead>`
+-   :ref:`iproto.threads <configuration_reference_iproto_threads>`
+
+
+..  _configuration_reference_iproto_advertise:
+
+iproto.advertise.*
+~~~~~~~~~~~~~~~~~~
+
+The ``iproto.advertise.*`` configuration options specify URIs used to advertise the current instance to other cluster members or clients.
+These options accept an URI in the following formats:
+
+-   User :ref:`credentials <configuration_reference_credentials>` and an address: ``username@host:port`` or ``username:password@host:port``.
+
+-   User credentials: ``username@`` or ``username:password@``.
+    In this case, an advertise address is taken from :ref:`iproto.listen <configuration_reference_iproto_listen>`.
+
+-   An address: ``host:port``.
+
+If ``password`` is missing, it is taken from :ref:`credentials <configuration_reference_credentials>` for the specified ``username``.
+
+You can also use a Unix domain socket (``unix/:``) instead of ``host:port``.
+
+.. NOTE::
+
+    The ``host`` value cannot be ``0.0.0.0``.
+
+
+.. _configuration_reference_iproto_advertise_client:
+
+.. confval:: iproto.advertise.client
+
+    An URI used to advertise the current instance to clients.
+
+    | Type: string
+    | Default: box.NULL
+    | Environment variable: TT_IPROTO_ADVERTISE_CLIENT
+
+.. _configuration_reference_iproto_advertise_peer:
+
+.. confval:: iproto.advertise.peer
+
+    An URI used to advertise the current instance to other cluster members.
+    For example, it can be used by an instance to connect to another instance as a replica.
```

Comment:
I think 'can be used' is not really the same as 'enables super cool automatic that builds replicaset peer URIs for each instance' :)

I mean, the idea of the option is to let the automatic do the job. 'Can be used' means just that this information is present.

BTW, 'as a replica' is a bit inaccurate. The same URI list build logic works on a master too. I would use neural 'instance' or 'replicaset peer' in such contexts.

Reply:
Looks like we need to add links to the above section `iproto.advertise.*`, which describes all the details common for all the options.